lib/commons/text/accessible-text-virtual.js

```diff
@@ -60,11 +60,11 @@ function accessibleTextVirtual(virtualNode, context = {}) {
  * @param {VirtualNode} element
  * @return {String} textContent value
  */
-function textNodeContent({ actualNode }) {
-	if (actualNode.nodeType !== 3) {
+function textNodeContent(virtualNode) {
+	if (virtualNode.props.nodeType !== 3) {
 		return '';
 	}
-	return actualNode.textContent;
+	return virtualNode.props.textContent;
```

Comment:
I think nodeValue is more appropriate here. In this particular case it does the same thing, but what we're getting at here is the value of the text node, not the text content of the node and all its descendants, which is what textContent does.

```suggestion
	return virtualNode.props.nodeValue;
```